File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/viewrecent/fragment/presenter/ViewRecentPresenter.java
package ua.meugen.android.pdfcomposer.ui.activities.viewrecent.fragment.presenter;

import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.presenter.MvpPresenter;
import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.state.MvpState;


public interface ViewRecentPresenter extends MvpPresenter<MvpState> {

    void load();
}

File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/viewrecent/fragment/presenter/ViewRecentPresenter.kt
package ua.meugen.android.pdfcomposer.ui.activities.viewrecent.fragment.presenter

import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.presenter.MvpPresenter
import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.state.MvpState


interface ViewRecentPresenter : MvpPresenter<MvpState> {

    fun load()
}
